Packs/QualysVulnerabilityManagement/Integrations/QualysVulnerabilityManagement/QualysVulnerabilityManagement.py

```diff
@@ -8,297 +8,757 @@
-                                                  'recipient_group', 'ips', 'asset_group_ids', 'asignee_type'],
-                                         'api_route': API_SUFFIX + '/report/?action=launch&report_type=Remediation',
-                                         'call_method': 'POST', 'resp_type': 'text',
-                                         'json_path': "SIMPLE_RETURN.RESPONSE.ITEM_LIST.ITEM",
-                                         'table_name': 'Remediation Report Launch'},
-    'qualys-report-launch-compliance': {'context_prefix': 'Qualys.Report', 'context_key': 'ID',
-                                        'args': ['template_id', 'report_title', 'output_format',
-                                                 'hide_header', 'recipient_group_id', 'pdf_password',
-                                                 'recipient_group', 'ips', 'asset_group_ids', 'report_refs'],
-                                        'api_route': API_SUFFIX + 'report/?action=launch&report_type=Compliance',
-                                        'call_method': 'POST', 'resp_type': 'text',
-                                        'json_path': "SIMPLE_RETURN.RESPONSE.ITEM_LIST.ITEM",
-                                        'table_name': 'Compliance Report Launch'},
-    'qualys-report-launch-compliance-policy': {'context_prefix': 'Qualys.Report', 'context_key': 'ID',
-                                               'args': ['template_id', 'report_title', 'output_format',
-                                                        'hide_header', 'recipient_group_id', 'pdf_password',
-                                                        'recipient_group', 'ips', 'asset_group_ids', 'policy_id',
-                                                        'host_id', 'instance_string'],
-                                               'api_route': API_SUFFIX + '/report/?action=launch&report_type=Policy',
-                                               'call_method': 'POST', 'resp_type': 'text',
-                                               'json_path': "SIMPLE_RETURN.RESPONSE.ITEM_LIST.ITEM",
-                                               'table_name': 'Policy Report Launch'},
-    'qualys-virtual-host-manage': {'context_prefix': 'Qualys.VirtualEndpoint', 'context_key': 'DATETIME',
-                                   'args': ['action', 'ip', 'network_id', 'port', 'fqdn'],
-                                   'api_route': API_SUFFIX + 'asset/vhost/', 'call_method': 'POST', 'resp_type': 'text',
-                                   'json_path': "SIMPLE_RETURN.RESPONSE", 'table_name': ''},
-    'test-module': {'api_route': API_SUFFIX + '/scan/?action=list', 'call_method': 'POST', 'resp_type': 'text',
-                    'args': []},
-    'args_values': {}, 'inner_args_values': {}
+# Data for parsing and creating output
+commands_parse_and_output_data: Dict[str, Dict[Any, Any]] = {
```

Comment:
We use upper case names for constants, please change them.

Reply:
fixed